### ISSUE TITLE
Move Edit profile and logout buttons to the profile avatar

### ISSUE THREAD

**Issue by chris-t-li** [Fantasy-Fit/fantasy-fit-web#98]
![Screen Shot 2023-05-02 at 11 18 06 AM](https://user-images.githubusercontent.com/111921809/235710314-f1925c63-9939-41d3-8055-a7127540b1c9.png)

**Comment by dovid11564** [comment]
For clarification, is this the functionality you are looking for?
- When a user clicks on their profile icon, a dropdown will open up where they can interact with the buttons

As an aside, it looks like this functionality has already been implemented for the logout button.

**Comment by chris-t-li** [comment]
Correct. I moved the logout button from the profile section to a pop up that opens when you click the avatar at the top left corner. However, I also want to move the "edit profile" button to the same popup.

**Comment by chris-t-li** [comment]
For example, like this: 

<img width="377" alt="Screen Shot 2023-06-06 at 1 29 56 PM" src="https://github.com/Fantasy-Fit/fantasy-fit-web/assets/111921809/6ed2ef3e-6ff1-4f5c-978b-4b09686e7c07">

Note, I already provisioned for this in client/src/components/Header.jsx lines 46 - 50. Which is commented out right now.

```
        <button>
          <div>
            <EditIcon />Profile
          </div>
        </button>
```

And functionality would be, when you click, it opens up the EditProfileModal and hides the popup:

<img width="594" alt="Screen Shot 2023-06-06 at 1 30 02 PM" src="https://github.com/Fantasy-Fit/fantasy-fit-web/assets/111921809/dabbc4f3-7658-473c-825c-6f9491e0987d">

**Comment by dovid11564** [comment]
Cool. I'd like to take a crack at it. I'm going to assign this to myself.
